### PR TITLE
Start Align from HF + pass compute through align

### PR DESCRIPTION
**Pull request by Jacobsolawetz**

### CODE REVIEW

File: arcee/api.py [hunk 1]
```diff
@@ -292,6 +292,9 @@ def start_alignment(
     pretrained_model: Optional[str] = None,
     merging_model: Optional[str] = None,
     alignment_model: Optional[str] = None,
+    hf_model: Optional[str] = None,
+    target_compute: Optional[str] = None,
+    capacity_id: Optional[str] = None,
```
Comment (shahrzads) [inline arcee/api.py:297]: Adding a brief description for each of these args could be helpful and make the function definition structure more consistent.

**Review by shahrzads — APPROVED** [review]

**Comment by shahrzads** [comment]
Added a quick suggestion, but I believe the changes are fine.

**Review by tleyden — APPROVED** [review]
LGTM but it looks like you need to rebase on main

**Review by tleyden — APPROVED** [review]
LGTM

File: arcee/api.py [hunk 2]
```diff
@@ -377,14 +378,11 @@ def start_alignment(
...
+        "target_compute": target_compute,
+        "capacity_id": capacity_id,
     }
 
-    if target_compute:
-        data["target_compute"] = target_compute
-
-    if capacity_id:
-        data["capacity_id"] = capacity_id
-
```
[the remark this answers is not in the document]
Comment (Jacobsolawetz) [reply arcee/api.py:387]: Passed em all in the json

[the remark this answers is not in the document]
Comment (Jacobsolawetz) [reply arcee/api.py:387]: So yes!

[the remark this answers is not in the document]
Comment (tleyden) [reply arcee/api.py:387]: Ah I see it now

**Pull request merged**